.github/actions/common/artifact_utils.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import annotations
+
+import argparse
+import os
+from pathlib import Path
+
+
+def add_common_args(parser: argparse.ArgumentParser):
+    parser.add_argument('-s', '--commit_sha', help='Commit hash for which artifacts were generated', required=True)
+    parser.add_argument('-b', '--branch_name', help='Name of GitHub branch', required=False,
+                        default=os.getenv('GITHUB_BASE_REF') or os.getenv('GITHUB_REF_NAME'))
+    parser.add_argument('-e', '--event_name', help='Name of GitHub event', required=False,
+                        default=os.getenv('GITHUB_EVENT_NAME'))
+    parser.add_argument('--storage_dir', help='Subdirectory name for artifacts, same as product type', required=True)
+    parser.add_argument('--storage_root', help='Root path of the artifacts storage', required=False,
+                        default=os.getenv('ARTIFACTS_SHARE'))
+
+
+def get_event_type(event_name: str = os.getenv('GITHUB_EVENT_NAME')) -> str:
+    return 'pre_commit' if event_name == 'pull_request' else 'commit'
```

Comment:
Create a `constants` file for such magic strings? + maybe create an enum for these ones.